parts/kubernetesmastercustomdata.yml

```diff
@@ -308,6 +310,10 @@ write_files:
     [Service]
     Restart=always
     ExecStartPre=/bin/mkdir -p /var/lib/kubelet
+    # Azure does not support two LoadBalancers(LB) sharing the same nic and backend port.
+    # As a workaround, the Internal LB forwards apiserver traffic from port 4443 to 443 while the 
```

Comment:
I don't think the comment is accurate ? (or else I'm reading it wrong...)

I thought it went like this: `ILB:443 -> Node:4443 -> iptables -> internal:443`